pkg/kubelet/cadvisor.go

```diff
@@ -24,6 +25,17 @@ import (
 	cadvisor "github.com/google/cadvisor/info"
 )
 
+var (
+	// ErrNoKubeletContainers returned when there are not containers managed by the kubelet (ie: either no containers on the node, or none that the kubelet cares about).
+	ErrNoKubeletContainers = errors.New("No containers managed by kubelet")
```

Comment:
I'm following a pattern in dockertools which uses errors this way and defines them with starting caps. Currently running e2e, let me know if you'd like me to change it otherwise I plan to leave it as is.